server/gameData_test.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetCardCount(t *testing.T) {
+	//TODO: Setup mock database
```

Comment:
Ideally, after the changes Matt made, the MockDB struct should already be preloaded to return the data you need for your unit tests and shouldn't need any `setup`. It's selected by default on unit tests. You'll just need to check validate your tests according to what it returns.

Reply:
I can't figure out how to get the mock database to work. 

Here is how I am setting up the database:
```
	database, err := db.GetDb()
	assert.NoError(t, err)

	game, err := database.CreateGame()
	assert.NoError(t, err)

	player, err := database.CreatePlayer("Player 1")
	assert.NoError(t, err)

	err = database.JoinGame(game.ID, player.ID)
	assert.NoError(t, err)
```

However, when I call `database.LookupGameByID(gameID)` I always get ` &{5f1e42344170d7831aeaec08  [] [] [] 0  false}` as a result (note that the UUID does change but the rest remains the same, namely, unpopulated).

I originally thought that there was something wrong with the mock, and indeed, `JoinGame` doesn't seem to do anything. My best guess as to an implementation is
```
func (db *mockDB) JoinGame(id string, username string) error {
	db.games[id].Players[0] = db.players[username]
	return nil
}
```
Yet I'm not getting anything different from the lookup, so I'm stumped and confused.

Reply:
@Aaron-Fine I found the issue with JoinGame and pushed up changes to the MockDB.  You should be able to test this now. 

You were correct. that particular endpoint was a stub. Let me know if you face errors now.